cpp/HelloWorld/client/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.22)
+project(armonik_cpp_hello_client VERSION 0.1 LANGUAGES C CXX)
+
+find_package(Protobuf REQUIRED)
+find_package(gRPC CONFIG REQUIRED)
+find_package(Threads)
+find_package(ArmoniK.Api.Common REQUIRED)
+find_package(ArmoniK.Api.Client REQUIRED)
```

```suggestion
find_package(Threads)
find_package(ArmoniK.Api.Client REQUIRED)
```
The client package has all the other packages in dependency. So you get them by transitivity